tox.ini

```diff
@@ -97,3 +97,12 @@ commands =
     rm {toxworkdir}/snyk
     rm {toxworkdir}/snyk-to-html-linux
     rm snyk-req.txt
+
+[testenv:bandit-scan]
+skip_install = true
+deps =
+    bandit
+allowlist_externals =
+    bandit
+commands =
+    - bandit -r -c {toxinidir}/ipas_default.config {toxinidir}/ -f txt -o {toxworkdir}/bandit-report.txt
```

Comment:
Maybe we can place the config in `.ci`. We can then use `{toxinidir}/.ci/ipas_default.config`

Reply:
that's clear. will update.

Reply:
Same as the above comment. I think we should have something like 

```
bandit -r -c {toxinidir}/pyproject.toml ...
```